resolver/resolver.cc

```diff
@@ -4047,6 +4047,7 @@ void sanityCheck(const core::GlobalState &gs, vector<ast::ParsedFile> &trees) {
         ResolveSanityCheckWalk sanity;
         for (auto &tree : trees) {
             core::Context ctx(gs, core::Symbols::root(), tree.file);
+            ENFORCE(tree.tree);
```

Comment:
It seems weird to me that the earlier `ENFORCE` was checking the file and this is checking the tree, but 🤷‍♂️ .

Reply:
oh I didn't even notice that.

Reply:
I think something like `file.exists()` could only ever fire if we ran Stripe's codebase with debug checks—it's abundantly unlikely for us to write a test case that would trip that in e.g. a `test/testdata/`-style test. But I could see the nullptr `tree.tree` enforce firing, so I'm going to leave it as is.